libmatekbd/matekbd-indicator.c

```diff
@@ -414,10 +410,7 @@ matekbd_indicator_prepare_drawing (MatekbdIndicator * gki, int group)
 			ln2cnt_map = NULL;
 		}
 
-		gtk_container_add (GTK_CONTAINER (align), label);
-		gtk_container_add (GTK_CONTAINER (ebox), align);
-
-		gtk_container_set_border_width (GTK_CONTAINER (align), 2);
```

Comment:
Normal, i would set everything from the GtkAlignment to his parent, but the parent is a eventbox.
And in gtk-docs i found nothing to set a border on a eventbox.
Any idea?